documentation/src/main/asciidoc/topics/rest_examples/rest_full_cache_detail.json

```diff
@@ -0,0 +1,48 @@
+{
+  "stats": {
+    "misses": -1,
+    "timeSinceStart": -1,
+    "timeSinceReset": -1,
+    "hits": -1,
+    "currentNumberOfEntries": -1,
+    "currentNumberOfEntriesInMemory": -1,
+    "totalNumberOfEntries": -1,
+    "stores": -1,
+    "offHeapMemoryUsed": -1,
+    "dataMemoryUsed": -1,
+    "retrievals": -1,
+    "removeHits": -1,
+    "removeMisses": -1,
+    "evictions": -1,
+    "averageReadTime": -1,
+    "averageReadTimeNanos": -1,
+    "averageWriteTime": -1,
+    "averageWriteTimeNanos": -1,
+    "averageRemoveTime": -1,
+    "averageRemoveTimeNanos": -1,
+    "requiredMinimumNumberOfNodes": -1
+  },
+  "size": 0,
+  "configuration": {
+    "template": false,
+    "elementDefinition": {
+      "topLevel": true
+    }
+  },
+  "members": [
```

Comment:
There is some misalignment with the information presented in the cache manager resource
https://infinispan.org/docs/dev/titles/rest/rest.html#rest_v2_cache_manager_info

For example, there we have
```json
 "cluster_members":[
        "CacheManagerResourceTest-NodeA-49696",
        "CacheManagerResourceTest-NodeB-28120"
 ],
```

while here it's completely different. It's better to reuse POJOs to make the information consistent across endpoints if we are going to duplicate some of the info...

Reply:
totally agree